tests/integration/query/commits/with_doc_id_cid_test.go

```diff
@@ -70,14 +71,15 @@ func TestQueryCommitsWithDocIDAndCidForDifferentDocWithUpdate(t *testing.T) {
 				Request: ` {
 						commits(
 							docID: "bae-not-this-doc",
-							cid: "bafybeica4js2abwqjjrz7dcialbortbz32uxp7ufxu7yljbwvmhjqqxzny"
+							cid: "bafyreiale6qsjc7qewod3c6h2odwamfwcf7vt4zlqtw7ldcm57xdkgxja4"
```

Comment:
just highlighting this for reviewers. This isnt required by the new changes to the commit query semantics. Its actually fixing a unconsequential bug in using the incorrect `CID` for the goal of the test. The test wanted to test field IDs, but the original `CID` was a composite `CID`.